.github/workflows/push-trigger.yml

```
@@ -14,16 +14,39 @@ on:
-      - '!release-branch'
-      - master
-      - 1.*
-      - develop
-      - MOSIP*
-      - release*
-      - ES-842
-
+      - "**"
+      
 jobs:
+# Detect which parts of the repo have changed
```

actionlint (1.7.8)

could not parse as YAML: did not find expected key

(syntax-check)